### ISSUE TITLE
more options for save filenames

### ISSUE THREAD

**Issue by sparr** [andymeneely/squib#203]
It would be nice to have more control over the names of multiple saved files from save_png.

Currently I can fake a filename suffix by doing something like `prefix: 'PREFIX', count_format: '%02dSUFFIX' but it would be nice to be able to do that directly.

More generally, I'd love to just be able to provide an array of filenames, the same way an array is provided for most of the other methods, something like `save names: ['001', 'secondfile', 'C'], format: :png`, which would allow putting the names together with a fully capable block of ruby elsewhere and not needing this sort of ugliness:

```
for i in 0..filenames.length-1
  save prefix: filenames[i], range: [i], count_format: '', format: :png
end
```

**Comment by andymeneely** [comment]
Yeah we were talking about this in #159. Definitely on my radar. I'm doing some refactoring on how arguments work that would make this a much easier change than it would be now. But for now just do it with count_format.
